src/tools/rbd/CMakeLists.txt

```
@@ -68,6 +69,14 @@ target_link_libraries(rbd
   librados
   ceph-common global
   ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS})
+if(WITH_UBBD)
```

I would add RBD to the name to make it clear what it's about:
```suggestion
if(WITH_RBD_UBBD)
```

---

Totally agreed, I missed it